ui/app/rif/components/searchDomains.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import Select from "react-dropdown-select"
+import rifActions from '../../rif/actions'
+
+class SearchDomains extends Component {
+	state = {
+		domains: [],
+	}
+	constructor(props) {
+		super(props);
+		if(localStorage.rnsDomains){
+			let domains = JSON.parse(localStorage.rnsDomains);
+			this.state = { 
+				domains: domains,
+			}
+		}	
+	}
+  render () {
+	return (
+		<Select 
```

Comment:
Why are we using a select instead of a searchbar?

Reply:
Now its a input type. Check it out please!